x-pack/platform/plugins/private/translations/translations/fr-FR.json

```diff
@@ -7503,32 +7501,6 @@
     "searchErrors.search.httpErrorTitle": "Impossible de se connecter au serveur Kibana",
     "searchErrors.tsdbError.message": "Le champ {field} du type de série temporelle [compteur] a été utilisé avec une opération {op} non prise en charge.",
     "searchErrors.tsdbError.tsdbCounterDocsLabel": "Pour en savoir plus sur les types de champs des séries temporelles et les agrégations compatibles [counter]",
-    "xpack.searchIndexDocuments.documentList.description": "Affichage de {results} sur {total}. Nombre maximal de résultats de recherche de {maximum} documents.",
```

Comment:
Could you please double check these removed translations? I don't know why it was done, but probably we should remove only related to on/off